container/raw/handler.go

```diff
@@ -227,6 +227,9 @@ func (self *rawContainerHandler) getFsStats(stats *info.ContainerStats) error {
 }
 
 func (self *rawContainerHandler) GetStats() (*info.ContainerStats, error) {
+	if *disableRootCgroupStats && isRootCgroup(self.name) {
```

Comment:
can we do this in CanHandleAndAccept in factory.go similar to the `docker_only` flag?

Reply:
Sure will test it and update you.

Reply:
@dashpole :

We cant do it in "CanHandleandAccept" currently as [this step](https://github.com/google/cadvisor/blob/master/manager/manager.go#L273) throws error that it failed to find the container ("/") as the [container creation step for "/"](https://github.com/google/cadvisor/blob/master/manager/manager.go#L268) skips creating the container with message 

> 

I0810 00:05:34.490103  103510 factory.go:170] Factory "raw" can handle container "/", but ignoring.
I0810 00:05:34.490112  103510 manager.go:908] ignoring container "/"

>

Reply:
Ok.  Thanks for looking into it.